serverless/messaging/api-cli/connect-aws-cli.mdx

```diff
@@ -77,11 +77,13 @@ Now you have installed the AWS-CLI, you need to configure it for use with your S
     <Message type="important">
       If you are using the AWS-CLI v2, you must include the path to the plugin in your configuration file. Add `cli_legacy_plugin_path = <path-to-plugin>` to the `[plugins]` section, replacing `<path-to-plugin>` with the corresponding path.
     </Message>
+
 3. Generate a credentials file using the following command:
     ```
     aws configure
     ```
-4. Open the `~/.aws/credentials` file you just created, and add the access key and secret key you saved when you generated your [SQS/SNS namespace's credentials](/serverless/messaging/how-to/create-credentials/) as follows:
+
+4. Open the `~/.aws/credentials` file you just created, and add the access key and secret key you saved when you generated your [SQS or SNS credentials](/serverless/messaging/how-to/create-credentials/) as follows:
```

Comment:
Since there are now separate credentials for SQS and SNS, how should we deal with this step?

Reply:
2 credentials need to be created and added to the `credentials` file:

```
[sns]
aws_access_key_id=<ACCESS_KEY_FOR_SNS>
aws_secret_access_key=<SECRET_KEY_FOR_SNS>

[sqs]
aws_access_key_id=<ACCESS_KEY_FOR_SQS>
aws_secret_access_key=<SECRET_KEY_FOR_SQS>
```